packages/icons/src/Svg.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Box, system } from '@marigold/system';
+
+type SvgProps = {
+  size?: number;
+  fill?: string;
+};
+
+export const Svg = system<SvgProps, 'svg'>(
+  ({ size = 24, fill = 'currentcolor', children, ...props }) => {
+    return (
+      <Box
+        as="svg"
+        xmlns="http://www.w3.org/2000/svg"
+        width={size}
+        height={size}
+        viewBox="0 0 100 100"
```

Comment:
Why 100?

Reply:
At "0 0 24 24" three quarters of the icon are cut off. At "0 0 100 100" it is displayed completely. I don't understand this either, because "0 0 24 24" should work

Reply:
Are all Icons of this size? This has to be the same all the time then. If so, do they have to be that detailed? CC @Toshibot

Reply:
Its working now, thank you!